Legacy/real_time/arduino_src/radio_buggy_mega/Makefile

```diff
@@ -467,51 +489,46 @@ extcoff: $(TARGET).elf
 
 
 # Link: create ELF output file from object files.
-.SECONDARY : $(TARGET).elf

```

Comment:
Do we not care about keeping the ELF files anymore? If not, also delete the comment